proxy/src/main/java/com/velocitypowered/proxy/protocol/packet/Respawn.java

```diff
@@ -13,17 +14,21 @@
   private short difficulty;
   private short gamemode;
   private String levelType = "";
+  private boolean shouldKeepPlayerData; // 1.16+
+  private DimensionInfo dimensionInfo;
 
   public Respawn() {
   }
 
   public Respawn(int dimension, long partialHashedSeed, short difficulty, short gamemode,
-      String levelType) {
```

Comment:
Should I also make a dummy constructor here @astei ?

Reply:
The packet API is internal details. You can create a constructor for compatibility if you want, but it won't prevent me from merging this in.